pkg/trivy/wrapper_test.go

```diff
@@ -23,111 +22,86 @@ const (
-	}]
-},
-{
-	"Target": "php-app/composer.lock",
-	"Vulnerabilities": [{
-		"VulnerabilityID": "CVE-2016-5385",
-		"PkgName": "guzzlehttp/guzzle",
-		"InstalledVersion": "6.2.0",
-		"FixedVersion": "6.2.1",
-		"Severity": "MEDIUM",
-		"References": [
-			"http://linux.oracle.com/cve/CVE-2016-5385.html"
-		],
-		"LayerID": "sha256:5216338b40a7b96416b8b9858974bbe4acc3096ee60acbc4dfb1ee02aecceb11"
-	}]
-},
-{
-	"Target": "python-app/Pipfile.lock",
-	"Vulnerabilities": null
-}
-]`
-)
-
-func TestWrapperParseScanReports(t *testing.T) {
-	testCases := []struct {
-		name           string
-		jsonOutput     string
-		expectedReport ScanReport
-		expectedError  error
-	}{
-		{
-			name:       "Should parse scan report with application dependencies",
-			jsonOutput: sampleReport,
-			expectedReport: ScanReport{
-				Target: "knqyf263/vuln-image (alpine 3.7.1)",
-				Vulnerabilities: []Vulnerability{
-					{
-						VulnerabilityID:  "CVE-2018-6543",
-						PkgName:          "binutils",
-						InstalledVersion: "2.30-r1",
-						FixedVersion:     "2.30-r2",
-						Severity:         "MEDIUM",
-						References: []string{
-							"https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-6543",
-						},
-						LayerID: "sha256:5216338b40a7b96416b8b9858974bbe4acc3096ee60acbc4dfb1ee02aecceb10",
-					},
-					{
-						VulnerabilityID:  "CVE-2019-11358",
-						PkgName:          "jquery",
-						InstalledVersion: "3.3.9",
-						FixedVersion:     "3.4.0",
-						Severity:         "MEDIUM",
-						References: []string{
-							"http://packetstormsecurity.com/files/152787/dotCMS-5.1.1-Vulnerable-Dependencies.html",
-							"http://packetstormsecurity.com/files/153237/RetireJS-CORS-Issue-Script-Execution.html",
-						},
-						LayerID: "sha256:5216338b40a7b96416b8b9858974bbe4acc3096ee60acbc4dfb1ee02aecceb10",
-					},
-					{
-						VulnerabilityID:  "CVE-2016-5385",
-						PkgName:          "guzzlehttp/guzzle",
-						InstalledVersion: "6.2.0",
-						FixedVersion:     "6.2.1",
-						Severity:         "MEDIUM",
-						References: []string{
-							"http://linux.oracle.com/cve/CVE-2016-5385.html",
-						},
-						LayerID: "sha256:5216338b40a7b96416b8b9858974bbe4acc3096ee60acbc4dfb1ee02aecceb11",
-					},
+}]`
+	expectedReport = ScanReport{
+		Target: "alpine:3.10.2",
+		Vulnerabilities: []Vulnerability{
+			{
+				VulnerabilityID:  "CVE-2018-6543",
+				PkgName:          "binutils",
+				InstalledVersion: "2.30-r1",
+				FixedVersion:     "2.30-r2",
+				Severity:         "MEDIUM",
+				References: []string{
+					"https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-6543",
 				},
+				LayerID: "sha256:5216338b40a7b96416b8b9858974bbe4acc3096ee60acbc4dfb1ee02aecceb10",
 			},
 		},
-		{
-			name:          "Should return error when scan report is empty",
-			jsonOutput:    emptyReport,
-			expectedError: xerrors.New("expected at least one report"),
-		},
+	}
+)
+
+func TestWrapper_Scan(t *testing.T) {
```

Comment:
This is awesome. Should we add some sad path tests?

Reply:
I wanted to ship #82 ASAP, but yes, we should increase branch coverage for wrapper.go in the subsequent PRs.